src/components/pages/homepage/press-logos.module.scss

```diff
@@ -8,5 +8,6 @@
     filter: grayscale(1);
     height: 40px;
     margin: 1rem 2rem;
+    margin: 3rem;
```

Comment:
Hi, @karinanguyen, and thanks for your first PR! I think we shouldn't have two `margin` declarations in here.

Reply:
So sorry, thank you for the catch ! Just edited:)